src/components/TractionHeader.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+    <div :class="headerClasses">
+      <div class="max-w-7xl mx-auto px-4 sm:px-6">
+        <div class="flex justify-between items-center py-4 md:justify-start md:space-x-10">
+          <div class="flex flex-row gap-x-2">
+            <img class="w-8 h-8" src="../images/traction-logo.svg" alt="Traction logo" />
+            <div class="text-white text-2xl">Traction</div>
+            <TractionLink name="Home" link="/dashboard" view-type="2" />
+            <TractionLink name="Label Printing" link="/label-printing" view-type="2" />
+            <TractionLink name="QC Results Upload" link="/qc-results-upload" view-type="2" />
+            <TractionLink name="Reception" link="/reception" view-type="2" />
+          </div>
+        </div>
+      </div>
+    </div>
+</template>
+
+<script>
+    import TractionLink from '@/components/TractionLink'
+    
+    export default {
+      components: {
+         TractionLink
+      },
+      name: 'TractionHeader',
+      data(){
+        return {
+           environment: process.env.NODE_ENV,
+        };
+      },
+      computed:{
+        headerClasses(){
+            let bgColorClass = '';
+            if (this.environment == 'development'){
+                bgColorClass = 'bg-blue-600';
+            }else if(this.environment == 'staging'){
```

Comment:
Is there no 'uat' environment or is the 'uat' environment called as 'staging'?

Reply:
In fact, I am not sure what uat environment will return.  Will it only be known when deployed in 'uat'? 
I actually guess it might just return to be 'production' in uat.

Reply:
> In fact, I am not sure what uat environment will return. Will it only be known when deployed in 'uat'? I actually guess it might just return to be 'production' in uat.

We might need to add an environment specific variable that we add to the .env file and the deployment project e.g.

VITE_ENVIRONMENT = development

Reply:
Yes, I think so, I think it should be uat for uat environment. @stevieing please correct if I am wrong.

Reply:
good point!